src/app/components/common/lineChart/lineChart.service.ts

```diff
@@ -347,7 +347,7 @@ export class LineChartService {
         });
 
         let divideBy: number;
-        let title: string = prop;
+        let title: string = prop == "ctl-tpc" ? "SCSI Target Port (tpc)" : prop; // Put in ugly override. Wasn't really a better place for this one change.
```

Comment:
I am not sure how other target ports named there in the new UI, but if there is some flexibility, it would be nice to decode the "tpc" here as Third-Party Copy, i.e. traffic caused by XCOPY operations.